src/activity_handlers.cpp

```diff
@@ -2042,14 +2042,16 @@ void activity_handlers::reload_finish( player_activity *act, player *p )
     item &ammo = *act->targets[ 1 ];
     std::string reloadable_name = reloadable.tname();
     std::string ammo_name = ammo.tname();
```

Comment:
Adding a comment to explain why these values are stored here would be a good idea, so no one tries to "simplify" the code by moving these checks back to the if statements in the future.